website/docs/cluster-management/managing-clusters-without-capi.mdx

```diff
@@ -82,28 +82,24 @@ kubectl create secret generic demo-01-kubeconfig \
 
 	Once we know what namespaces are available we can test whether the logged in user can access them via impersonation.
 
-1. Get the token of the service account
-
-	First get the list of secrets of the service accounts by running the following command:
+3. Retrieve the token from the service account. First, run this command to get the list of secrets of the service accounts:
 
-	```sh
+	```bash
 	kubectl get secrets --field-selector type=kubernetes.io/service-account-token
 	NAME                      TYPE                                  DATA   AGE
 	default-token-lsjz4       kubernetes.io/service-account-token   3      13d
 	demo-01-token-gqz7p       kubernetes.io/service-account-token   3      99m
 	```
 
-	`demo-01-token-gqz7p` is the secret that holds the token for `demo-01` service account
+	(`demo-01-token-gqz7p` is the secret that holds the token for `demo-01` service account.)
 
-	To get the token of the service account run the following command:
+	Then, run the following command to get the service account token:
 
-	```sh
+	```bash
 	TOKEN=$(kubectl get secret demo-01-token-gqz7p -o jsonpath={.data.token} | base64 -d)
 	```
 
-1. Create a kubeconfig secret
-
-	We'll use a helper script to generate the kubeconfig, save this into `static-kubeconfig.sh`:
+4. Create a kubeconfig secret. We'll use a helper script to generate the kubeconfig, and then save it into `static-kubeconfig.sh`:
```

Comment:
This is a bit awkard to read. We provide a helper script which the reader/user needs to save locally before they can use it.